gulpfile.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+/**
+  * @file
+  * This file contains gulp configurations for setting up SASS with feature of
+  * importing Shoreditch Partials and minifying the css file to .min.css
+  * Tasks
+  * default : Runs SASS task
+  * sass:sync : Runs sass sync task for civicrmScssRoot
+  * sass: Compiles civicase.scss under scss folder to CSS counterpart
+  * watch: Watches for scss file changes and run sass task
+  */
+
+// Gulp related variables
+const gulp = require('gulp');
+const bulk = require('gulp-sass-bulk-import');
+const sass = require('gulp-sass');
+const postcss = require('gulp-postcss');
+const postcssPrefix = require('postcss-prefix-selector');
+const postcssDiscardDuplicates = require('postcss-discard-duplicates');
+const stripCssComments = require('gulp-strip-css-comments');
+const transformSelectors = require('gulp-transform-selectors');
+const cssmin = require('gulp-cssmin');
+const rename = require('gulp-rename');
+const sourcemaps = require('gulp-sourcemaps');
+
+// Local variables
```

Comment:
All variables in this files are local. So this comment does not really make sense. I would say no need to comment `// Local variables` and `// Gulp related variables`, its quite evident by looking at them. An empty line between them is enough(as you have it).